Index: src/main/java/com/fastcampus/gearshift/controller/BAdminController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/fastcampus/gearshift/controller/BAdminController.java b/src/main/java/com/fastcampus/gearshift/controller/BAdminController.java
new file mode 100644
--- /dev/null	(date 1742272506044)
+++ b/src/main/java/com/fastcampus/gearshift/controller/BAdminController.java	(date 1742272506044)
@@ -0,0 +1,18 @@
+package com.fastcampus.gearshift.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+public class BAdminController {
+
+
+
+    @GetMapping("/admin/{page}")
+    public String loadAdminPage(@PathVariable("page") String page, Model model) {
+        // 유효한 JSP 파일인지 확인 후 반환
+        return "admin/" + page;
+    }
+}
+
